README.md

```diff
@@ -1,46 +1,95 @@
 # Jade Service Runner - A JSON-RPC Service Manager
 
-<p align="center">
-<img height="256" width="256" src="https://github.com/etclabscore/jade-media-assets/raw/master/jade-logo-light/jade-logo-light%20(PNG)/256x256.png" />
-</p>
-
-<center>
+<!-- project shields -->
   <span>
     <img alt="CircleCI branch" src="https://img.shields.io/circleci/project/github/etclabscore/jade-service-runner/master.svg">
     <img src="https://codecov.io/gh/etclabscore/jade-service-runner/branch/master/graph/badge.svg" />
     <img alt="Dependabot status" src="https://api.dependabot.com/badges/status?host=github&repo=etclabscore/jade-service-runner" />
-    <img alt="Chat on Discord" src="https://img.shields.io/badge/chat-on%20discord-7289da.svg" />
+    <img alt="Chat on Discord" src="https://img.shields.io/badge/chat-on%20discord-7289da.svg?link=https://discord.gg/S9AT3X2" />
```

Comment:
this should be wrapped in an `a` tag right? not working as link in the preview for me

Reply:
I've updated the discord badge. Works now.